lib/tessel/access-point.js

```diff
@@ -46,14 +47,46 @@ Tessel.prototype.enableAccessPoint = function() {
 Tessel.prototype.disableAccessPoint = function() {
   var status = 'Access Point successfully disabled.';
 
-  return new Promise((resolve) => {
+  return new Promise((resolve, reject) => {
     return this.simpleExec(commands.turnAccessPointOff())
       .then(() => {
-        return commitAndClose(this, status, resolve);
+        return commitAndClose(this, status, resolve, reject);
       });
   });
 };
 
+Tessel.prototype.getAccessPointInfo = function() {
+  var info;
+
+  return new Promise((resolve, reject) => {
+    return this.simpleExec(commands.getAccessPointConfig())
+      .then((config) => {
+        const props = ['ssid', 'key', 'encryption', 'disabled'];
```

Comment:
I think that `const` should be preferred whenever you have a variable that won't change.